routes/getDinoImage.js

```diff
@@ -11,9 +11,13 @@ const options = {
 const url = 'https://duckduckgo-image-search.p.rapidapi.com/search/image?q=dinosaur';
 
 router.get('/', async(request, response) => {
+  try {
     const fetchAPI = await fetch(url, options);
     const dinoImageResponse = await fetchAPI.json();
     response.json(dinoImageResponse);
+  } catch(error){
+    response.status(500).json({message: error.message});
```

Comment:
Added handling for catching errors for grabbing a dino image. This helps with the web app not crashing